src/editors/string.js

```diff
@@ -213,6 +213,10 @@ export class StringEditor extends AbstractEditor {
       /* it will generate an error trying to append it to the missing parentNode */
       if (this.input.parentNode) this.afterInputReady()
       if (this.adjust_height) this.adjust_height(this.input)
+      const output = this.control.querySelector('output')
+      if (output) {
```

Comment:
Are there cases when `output` is empty?
Would
```
this.control.querySelector('output').value = this.input.value
```
be sufficient?

Reply:
When string has format 'range' there is always an output. But a string doe not have to have always format 'range'

Reply:
Question: Wouldn't it be better to check for the `format` rather than an element `output`?

Reply:
I think checking both is better. Should i update?